avocado/utils/base_utils.py

```diff
@@ -0,0 +1,61 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+# Author: Santhosh G <santhog4@linux.vnet.ibm.com>
+#
+# This code was inspired in the autotest project,
+# client/shared/base_utils.py
+
+import os
+import re
+
+
+def check_kernel_ver(ver):
```

Comment:
Instead of this function one could use `assert LooseVersion(os.uname()[2]) > LooseVersion(ver)`. Anyway I see a benefit in having this part of the API for convenience, I'd just use the `distutils` here instead of custom code.